src/terminal/clear.lua

```diff
@@ -0,0 +1,113 @@
+--- Clear Module
+-- Provides functions to clear various parts of the terminal screen.
+-- @module clear
+
+local output = require "terminal.output"
+
+local M = {}
+
+-- Lazy loading terminal to avoid circular dependency
+--local terminal
+local function get_terminal()
+    return require("terminal")
+end
```

Comment:
this lazy loading can now be removed right?
```suggestion
local terminal = require "terminal"
local output = require "terminal.output"

local M = {}

```

Reply:
Done.